kickoff/dockerflow.py

```diff
@@ -0,0 +1,43 @@
+from os import path
+
+from flask import Response, jsonify
+from sqlalchemy.exc import OperationalError
+from kickoff import db
+
+
+# Wrapper that creates the endpoints required by CloudOps' Dockerflow spec: https://github.com/mozilla-services/Dockerflow
+# This gets used by both the admin and public apps.
+def create_dockerflow_endpoints(app):
+    HEADERS = {"Cache-Control": "no-cache"}
+
+    @app.route("/__version__")
+    def version():
+        version_file = app.config.get("VERSION_FILE")
+        if version_file and path.exists(version_file):
+            with open(version_file) as f:
+                version_json = f.read()
+            return Response(version_json, mimetype="application/json", headers={"Cache-Control": "no-cache"})
+        else:
+            return jsonify({
+                "source": "https://github.com/mozilla/balrog",
```

Comment:
Ouch! Shame on me!

That makes me wonder if I should hardcode the value here or read it from contribute.json anyway. What do you think?